tests/test_helpers.py

```
@@ -0,0 +1,362 @@
+    ----------
+    None
+        This test does not use pytest fixtures.
+
+    Returns
+    -------
+    None
+        The test passes when ``make help`` advertises the parent ``help`` and
+        ``test`` targets.
+    """
+    result = subprocess.run(
+        ["make", "help"],
```

Ruff (0.15.15)

[error] Starting a process with a partial executable path

(S607)